tests/e2e/test_spyre_max_new_tokens.py

```diff
@@ -38,7 +38,9 @@ def test_output(model: ModelInfo, stop_last: bool, max_model_len: int,
         logprobs=0,  # return logprobs of generated tokens only
         ignore_eos=False)
 
-    vllm_sampling_params = [vllm_sampling_params_normal] * 3
+    vllm_sampling_params = [
+        vllm_sampling_params_normal.clone() for _ in range(3)
+    ]
```

Comment:
This change is needed now that all tests use GTI by default (before this PR, only tests using `get_engine` would use GTI). Haivng a copied reference instead of a `.clone()` meant that all sequences had the same GTI config (even with different prompts).